processes/addFulfillManifest.py

```diff
@@ -33,51 +25,42 @@ def __init__(self, *args):
 
         # S3 Configuration
         self.s3Bucket = os.environ['FILE_BUCKET']
+        self.host = os.environ['DRB_API_HOST']
+        self.port = os.environ['DRB_API_PORT']
+        self.objKey = 'manifests/UofM/'
```

Comment:
I'd call this `prefix` or something. When its referenced in this class it gets confusing because in s3 keys usually refer to individual objects.

Reply:
Agreed. I'll change the name.